Externals/openal/OpenAL-Windows/Router/alc.cpp

```diff
@@ -1690,7 +1690,7 @@ ALCAPI ALboolean ALCAPIENTRY alcMakeContextCurrent(ALCcontext* context)
 
                 else
                 {
-                    alCurrentContext->Device->AlcApi.alcMakeContextCurrent(0);
+                    context->Device->AlcApi.alcMakeContextCurrent(0);
```

Comment:
At least try not to dereference `nullptr` here.

Reply:
Hmm...

Perhaps, @kcat can correct us if something is wrong here or it's ok? (oh, sorry, I know this code is very old😄)

Reply:
I think it should be correct.